src/Data/OpenApi/Internal.hs

```diff
@@ -838,8 +838,17 @@ data OAuth2Flows = OAuth2Flows
   , _oAuth2FlowsAuthorizationCode :: Maybe (OAuth2Flow OAuth2AuthorizationCodeFlow)
   } deriving (Eq, Show, Generic, Data, Typeable)
 
+data HttpSchemeType
+  = HttpSchemeBearer
+  | HttpSchemeBasic
+  deriving (Eq, Show, Generic, Data, Typeable)
+instance ToJSON HttpSchemeType where
```

Comment:
Please move these instances lower in the file, near instances for other security schemes.